packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1405,6 +1416,21 @@ export default class TradeStore extends BaseStore {
     get is_vanilla() {
         return this.contract_type === 'vanilla';
     }
+    async setContractPurchaseToastbox(response) {
```

Comment:
```suggestion
    setContractPurchaseToastbox(response) {
```

Reply:
no need for the async as not awaiting for anything.